File: api/src/shared/Venue.ts
import { Deal } from "./Deal"

export interface Venue {
    placeID: string
    address: string
    compoundID: string
    imageURL?: string
    name: string
    lat: number
    lng: number
    suburb?: string
    website?: string
    rating?: number
    priceLevel?: number
    types: string[]
}

export interface VenueDeals {
    venue: Venue
    deals: Deal[]
}

File: ios/LocalDeals/Model/Venue.swift
//  Created by Alexander Skorulis on 1/7/2022.

import Foundation

struct Venue: Codable {
    let website: String?
    let types: [String]
    let address: String
    let lng: Double
    let lat: Double
    let imageURL: String?
    let placeID: String
    let name: String
    let rating: Double
    let priceLevel: Int
    let compoundID: String
    
}
